packages/builders/scripts/orchestration/init-kitchen-sink.js

```diff
@@ -0,0 +1,25 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').CoreEvalBuilder} */
+export const defaultProposalBuilder = async ({ publishRef, install }) => {
+  return harden({
+    sourceSpec: '@agoric/orchestration/src/proposals/start-kitchen-sink.js',
+    getManifestCall: [
+      'getManifestForContract',
+      {
+        installKeys: {
+          kitchenSink: publishRef(
+            install(
+              '@agoric/orchestration/src/examples/kitchen-sink.contract.js',
+            ),
+          ),
+        },
+      },
+    ],
+  });
+};
+
+export default async (homeP, endowments) => {
+  const { writeCoreEval } = await makeHelpers(homeP, endowments);
+  await writeCoreEval('start-kitchen-sink', defaultProposalBuilder);
```

Comment:
I've been trying to reduce the cardinality of names for a core-eval script.

consider here re-using `startKitchenSink.name` by importing the starter function.

Reply:
> Hopefully, these can be codegen'ed

fwiw I think some helper functions will suffice

> should we consider publishing this builder

This open source repo is fully visible, and the docs site can point to it. Putting it in the NPM package creates an API to support